assets/src/components/admin/admin_tables.tsx

```diff
@@ -258,6 +258,19 @@ const BusShelterV2ScreensTable = (): JSX.Element => {
       disableFilters: true,
       FormCell: FormTextCell,
     },
+    {
+      Header: "Audio Interval Enabled",
```

Comment:
Maybe this could just be `Audio Interval?` — the only content of the data cells is the checkbox, so it's best if we can have the column take up less space.

Reply:
sounds good to me. Did you mean to include the `?`?

Reply:
Yep, since it's answering the "question" of whether the audio interval is enabled — like the Elixir convention to end booleans with a `?`. Probably not that important though.

Reply:
Cool, that's what I figured. Makes sense to me